### DISCUSSION TITLE
TikTok Login Kit Support

### DISCUSSION THREAD

**Discussion by OskarEichler** [cuongdevjs/reactjs-social-login#65] [Ideas]
Hey @cuongdevjs 

Would be awesome if you can look into also supporting TikTok:
https://developers.tiktok.com/doc/getting-started-create-an-app/

**Comment by cuongdevjs** [comment]
I will research to add it. Thank you

**Comment by cuongdevjs** [comment]
Tiktok login is supported, it will be released soon.

**Comment by cuongdevjs** [comment]
v2.6.0 has been released ✨

**Reply by OskarEichler** [reply]
Awesome thank you @cuongdevjs  🙏🏼
